### ISSUE TITLE
max_width not working

### ISSUE THREAD

**Issue by jbschrades** [imgix/imgix-rails#47]
I'm using this (HAML syntax):

```
ix_image_tag image, { w: 420, max_width: 540, auto: "format", fit: "crop", q: 70 }
```

And getting back this srcset:

``
<img max_width="540" srcset="https://host.imgix.net/uploads/post/266802f7-2fbd-44ab-9c38-d36dcaf819f1/Bill_Murray.jpg?ixlib=rails-2.1.4&amp;auto=format&amp;fit=max&amp;q=70&amp;w=100 100w, https://host.imgix.net/uploads/post/266802f7-2fbd-44ab-9c38-d36dcaf819f1/Bill_Murray.jpg?ixlib=rails-2.1.4&amp;auto=format&amp;fit=max&amp;q=70&amp;w=200 200w, https://host.imgix.net/uploads/post/266802f7-2fbd-44ab-9c38-d36dcaf819f1/Bill_Murray.jpg?ixlib=rails-2.1.4&amp;auto=format&amp;fit=max&amp;q=70&amp;w=300 300w, https://host.imgix.net/uploads/post/266802f7-2fbd-44ab-9c38-d36dcaf819f1/Bill_Murray.jpg?ixlib=rails-2.1.4&amp;auto=format&amp;fit=max&amp;q=70&amp;w=320 320w, https://host.imgix.net/uploads/post/266802f7-2fbd-44ab-9c38-d36dcaf819f1/Bill_Murray.jpg?ixlib=rails-2.1.4&amp;auto=format&amp;fit=max&amp;q=70&amp;w=400 400w, https://host.imgix.net/uploads/post/266802f7-2fbd-44ab-9c38-d36dcaf819f1/Bill_Murray.jpg?ixlib=rails-2.1.4&amp;auto=format&amp;fit=max&amp;q=70&amp;w=500 500w, https://host.imgix.net/uploads/post/266802f7-2fbd-44ab-9c38-d36dcaf819f1/Bill_Murray.jpg?ixlib=rails-2.1.4&amp;auto=format&amp;fit=max&amp;q=70&amp;w=420 420w, https://host.imgix.net/uploads/post/266802f7-2fbd-44ab-9c38-d36dcaf819f1/Bill_Murray.jpg?ixlib=rails-2.1.4&amp;auto=format&amp;fit=max&amp;q=70&amp;w=840 840w, https://host.imgix.net/uploads/post/266802f7-2fbd-44ab-9c38-d36dcaf819f1/Bill_Murray.jpg?ixlib=rails-2.1.4&amp;auto=format&amp;fit=max&amp;q=70&amp;w=1260 1260w" sizes="100vw" src="https://host.imgix.net/uploads/post/266802f7-2fbd-44ab-9c38-d36dcaf819f1/Bill_Murray.jpg?ixlib=rails-2.1.4&amp;auto=format&amp;fit=max&amp;q=70&amp;w=420" alt="Bill murray.jpg?ixlib=rails 2.1">
``

And then the 1260w gets delivered.  Is this expected behavior or am I doing something wrong? Trying to make the max width 540px.

**Comment by axelson** [comment]
I'm seeing the same problem, is there any resolution?

**Comment by joevandyk** [comment]
same here

**Comment by abrambailey** [comment]
Me too.

**Comment by sherwinski** [comment]
Hi everyone, apologies since this issue has been floating around for quite a while and never received a response. This looks like an issue that has long-been fixed since version `2.1.4`.

NB: The `max_width` option should be passed into the `tag_options` argument, such as in the following example:

```rb
<%= ix_image_tag('banner.jpg',
            url_params: { w: 420, auto: "format", fit: "crop", q: 70 },
            tag_options:{class:'banner-img', max_width: 540 }) %>
```

Of course, I can only speak to @jbschrades' specific case since they posted an example with the `imgix-rails` version number. I am going to close this ticket now, but if anyone else still has any further questions please feel free to write in to support@imgix.com.
